Gems/ScriptCanvasTesting/Code/Source/Framework/ScriptCanvasTestFixture.h

```diff
@@ -245,6 +257,71 @@ namespace ScriptCanvasTests
             EXPECT_EQ(m_graph->CanCreateConnectionBetween(targetEndpoint, sourceEndpoint).IsSuccess(), isValid);
         }
 
+        // Test if there is an existing connection between the provided endpoints
+        void TestIsConnectionBetween(const ScriptCanvas::Endpoint& sourceEndpoint, const ScriptCanvas::Endpoint& targetEndpoint, bool isValid = true)
+        {
+            AZ::Entity* ent;
+
+            EXPECT_EQ(m_graph->FindConnection(ent, sourceEndpoint, targetEndpoint), isValid);
+            EXPECT_EQ(m_graph->FindConnection(ent, targetEndpoint, sourceEndpoint), isValid);
```

Comment:
We can git rid of this second line because `FindConnection` is source/target agnostic.